cluster-install/install.sh

```diff
@@ -408,6 +523,27 @@ acp_is_enabled() {
     return 1
 }
 
+config_has_at_least_one_custom_registry_or_repo() {
```

Comment:
This check is used in tandem with checking if the `IMAGE_PULL_SECRET` is empty; if both are true, we prompt the user for a pull secret (or warn them if prompts are disabled).